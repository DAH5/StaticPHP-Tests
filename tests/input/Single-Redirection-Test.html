---
redirect: https://example.tld/new
---