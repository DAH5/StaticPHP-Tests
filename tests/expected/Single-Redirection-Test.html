<!DOCTYPE html>
<html lang="en">
<head>
	<meta http-equiv="refresh" content="0; url='https://example.tld/new'" />
	<title>Redirecting...</title>
</head>

<body>
	<p>If you are not redirected, <a href="https://example.tld/new">click here</a>.</p>
</body>
</html>
